metricbeat/templates/deployment.yaml

```diff
@@ -92,7 +92,11 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         - name: KUBE_STATE_METRICS_HOSTS
+          {{- if .Values.kube_state_metrics.enabled }}
           value: "$({{ .Release.Name | replace "-" "_" | upper }}_KUBE_STATE_METRICS_SERVICE_HOST):$({{ .Release.Name | replace "-" "_" | upper }}_KUBE_STATE_METRICS_SERVICE_PORT_HTTP)"
```

Comment:
Would it make more sense here to just use the kube_state_metrics_host value even if we are installing the sub-chart? In that case, what is the value that gets passed to the subchart as the service name?

Reply:
Hi @pbecotte . The general idea is to preserve upstream behavior as much as possible,
so when user disables `kube_state_metrics` installation then he/she needs to provide
existing endpoint for `kube_state_metrics`